cogs/welcome/welcome.py

```diff
@@ -132,6 +188,139 @@ async def logServerLeave(self, leaveUser: discord.Member):
     async def welcome(self, ctx: Context):
         """Server welcome message settings."""
 
+    @checks.mod_or_permissions()
+    @welcome.command(name="channel")
+    async def welcomeChannelSet(self, ctx, channel: discord.TextChannel):
+        """
+        Set the welcome channel
+
+        Parameters:
+        -----------
+        channel: The text channel to set welcome's to
+        """
+        if not isinstance(channel, discord.TextChannel):
+            return
+
+        await self.config.guild(ctx.guild).KEY_WELCOME_CHANNEL.set(channel.id)
+        await self.config.guild(ctx.guild).KEY_WELCOME_CHANNEL_SET.set(True)
+        await ctx.send(f"Channel set to {channel}")
+
+        return
+
+    @checks.mod_or_permissions()
+    @welcome.command(name="add")
+    async def greetAdd(self, ctx, name):
```

Comment:
please add type hints:
```suggestion
    async def greetAdd(self, ctx: Context, name: str):
```

Reply:
done now